cmake/module/GenerateHeaders.cmake

```diff
@@ -2,12 +2,19 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://opensource.org/license/mit/.
 
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.27)
+  set(DEPENDS_EXPLICIT_OPT DEPENDS_EXPLICIT_ONLY)
+else()
+  set(DEPENDS_EXPLICIT_OPT)
+endif()
+
```

Comment:
A note for reviewers. The Guix builds are using CMake 3.24.2 (the `cmake-minimal` package).